at-quannguyen/W4/controllers/session_controller.rb

```diff
@@ -0,0 +1,71 @@
+require_relative '../config/db'
+require 'bcrypt'
+$session = {}
+
+class SessionController
+  def initialize
+    @db = Db.new
+    @connect = @db.connect
+    @db.select_db
+  end
+
+  def login(obj)
+    if user_password_by_employee(obj.username, obj.password)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/7deb92b5e1c7275222ca44a289eebcc27cbc38fe/at-quannguyen/W4/controllers/session_controller.rb#L13"></span>Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression. (https://github.com/rubocop-hq/ruby-style-guide#no-nested-conditionals)</td>
    </tr>
  </tbody>
</table>